vendor/github.com/mitchellh/go-homedir/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
@alexellis, @stefanprodan, I don't like the fact that we checkin the code in vendor. I think I mentioned this before, but it should be that only use the vendor.conf to have the right versions, and then let the tool checkout everything before building. I would just put .gitignore in this folder.

Reply:
This is how dependencies are managed on Golang projects. It may seem odd, but you can find more information by looking at Docker or Google's Go projects.